### PR TITLE
Error when source directory does not exist.  Resolves #116.

### LINKED ISSUE #116: Compile with dirname fails silently when paths are wrong
Should fail with an OSError

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

File: sasstests.py [hunk 1]
```diff
@@ -846,6 +846,10 @@ def write_file(filename, contents):
 
 class CompileDirectoriesTest(unittest.TestCase):
 
+    def test_directory_does_not_exist(self):
+        with self.assertRaises(OSError):
+            sass.compile(dirname=('i_dont_exist_lol', 'out'))
```
Comment (dahlia) [inline sasstests.py]: According to the docs, use as a context manager of [`assertRaises()`][1] is introduced in Python 2.7.  See also [py26 build][2].

[1]: https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises
[2]: https://travis-ci.org/dahlia/libsass-python/jobs/106892363

Comment (asottile) [inline sasstests.py]: ah of course, I always forget this /me fixes

**Comment by asottile** [comment]
humm, seems mccabe 0.4.0 is not eggable in py35 on windows, @dahlia would you be opposed to abandoning `python setup.py test` and move to something like `py.test sasstests.py` ?

**Comment by dahlia** [comment]
No problem. I personally prefer py.test over unittest.

**Pull request merged**